hadoop-tools/hadoop-aws/src/test/java/org/apache/hadoop/fs/s3a/s3guard/ITestDynamoDBMetadataStore.java

```diff
@@ -471,7 +474,8 @@ public void testInitializeWithConfiguration() throws IOException {
     DynamoDBMetadataStore ddbms = new DynamoDBMetadataStore();
     try {
       ddbms.initialize(conf, new S3Guard.TtlTimeProvider(conf));
-      verifyTableInitialized(tableName, ddbms.getDynamoDB());
+      Table table = verifyTableInitialized(tableName, ddbms.getDynamoDB());
+      verifyTableSse(conf, table.getDescription());
```

Comment:
Please create a new test for this feature instead of extending one.

Reply:
In those tests we are not adding or changing the configuration. This is only to add more assert statements to test which ever SSE policy is configured, the test should pass.

I guess you are more concerned in the test we should have least have some default case where SSE is enabled? That way that code path will always get tested no matter what configuration is provided. I think I can add a test for that, borrowing or reusing most of the existing code...

Reply:
I'm happy with extending an existing one, given how long it takes to create and destroy a table